tests/clvm/test_puzzles.py

```diff
@@ -34,7 +34,7 @@
 
 def secret_exponent_for_index(index: int) -> int:
     blob = index.to_bytes(32, "big")
-    hashed_blob = BasicSchemeMPL.key_gen(std_hash(b"foo" + blob))
+    hashed_blob = AugSchemeMPL.key_gen(std_hash(b"foo" + blob))
```

Comment:
`key_gen` does not depend on the scheme. this is the only place in out code base where we refer to `BasicSchemeMPL`. Everywhere else we use `AugSchemeMPL`.